### ISSUE TITLE
[MVM] All players readying up does not start the wave if server is 5 or less

### ISSUE THREAD

**Issue by DrCactusTF2** [mastercomfig/team-comtress-2#270]
### Description
In MVM if you have a full party, every player hitting F4/Ready up will instantly start the 10 second count down.
But, if the server has 5 players or less, the timer will still think it is waiting for a non-existant 6th player to ready up.
This issue started happening around the time MYM was released I believe, as pre 2city you were able to instantly start a game with 5 people.

### Checklist

- [x] None of the open *or* closed issues document this problem.
- [x] This is a bug, not a new idea.
- [x] This is objectively a bug, and not something that people commonly disagree on.
- [x] This bug is in the code of the game, not in the textures, sounds, or other resources.
- [x] This issue occurs on the latest vanilla TF2 update.
- [ ] This issue is listed on the [Source 1 official bugtracker](https://github.com/ValveSoftware/Source-1-Games/issues).
- [ ] This issue occurs with DirectX level 100.
- [?] This issue occurs on the latest Team Comtress release, or the `master` branch. (need a team of 5 that could test this)
- [x] This issue occurs when using no custom content.
- [ ] This issue occurs when using `start_tf2.bat` (or `start_server.bat`) with no custom launch options.
- [ ] This issue occurs when using a listen server (using the `map` command ingame).
- [ ] This issue occurs when using a dedicated server.
- [ ] This bug is in the internal procedures or tooling (e.g. the build process).

### Reproduction
1. Acquire team of 5 in any MVM map
2. Everybody Ready up
3. Timer will go to 60 seconds, instead of 10

**Comment by joshuadinada** [comment]
If it's just one person it does ready up

**Comment by DrCactusTF2** [comment]
> 
> 
> If it's just one person it does start

Yeah, it seems to work in solo/duo, although  3-5 doesn't seem to work
https://youtu.be/LKZGtAChMa8

Here is a video of me testing this in live TF2 on a Valve Boot Camp server
